GUIDE.md

```diff
@@ -61,6 +62,32 @@ Once you deploy, you'll need to setup an admin user, whitelabel, and hostname be
 2. `heroku run --app=<your app> ./manage.py hostname <the hostname where you will run Shynet>`
 3. `heroku run --app=<your app> ./manage.py whitelabel "<your Shynet instance's name>"`
 
+## Render
+
+[Render](https://render.com) is a modern cloud platform to build and run all your apps and websites with free SSL, a global CDN, private networks and auto deploys from Git. Every Render service automatically includes:
+
+- No limits on simultaneous connections or connection duration.
+- No forced restarts.
+- Automatic [pull request previews](https://render.com/docs/pull-request-previews).
+- Infrastructure as Code with [`render.yaml`](https://render.com/docs/infrastructure-as-code).
+- [Fully managed PostgreSQL](https://render.com/docs/databases).
+- [Private networking](https://render.com/docs/private-services), load balancing, and service discovery.
+- HTTP health checks and [zero downtime deploys](https://render.com/docs/zero-downtime-deploys).
+- Free, unlimited [custom domains](https://render.com/docs/custom-domains) and [teams](https://render.com/docs/teams).
+- Persistent storage with [Render Disks](https://render.com/docs/disks).
```

Comment:
Could we remove this, as well as "Every Render service automatically includes:" from the sentence before? It reads too much like an ad, and doesn't contribute to the quality of the guide.